chainer/training/extensions/evaluator.py

```diff
@@ -148,6 +167,12 @@ def evaluate(self):
         accumulates the reported values to :class:`~chainer.DictSummary` and
         returns a dictionary whose values are means computed by the summary.
 
+        Note that this function assumes that the main iterator raises
+        ``StopIteration`` or some error is raised during the evaluation loop.
+        So, if the iterator does not raise ``StopIteratiron`` and
+        the loop does not raise no exceptions. It could be caught in the
```

Comment:
`...no exceptions. It could be...` -> `...no exceptions, it could be...`

Reply:
Fixed. I also modified points other than you pointed out. Could you read it again?